tests/integration/src/test/java/org/netxms/utilities/TestHelper.java

```diff
@@ -20,20 +20,34 @@
 
 import java.io.IOException;
 import java.util.Date;
+import java.util.HashMap;
+import java.util.Map;
 import org.netxms.client.NXCException;
 import org.netxms.client.NXCObjectCreationData;
+import org.netxms.client.NXCObjectModificationData;
 import org.netxms.client.NXCSession;
 import org.netxms.client.PollState;
 import org.netxms.client.ProtocolVersion;
+import org.netxms.client.datacollection.DataCollectionConfiguration;
+import org.netxms.client.datacollection.DataCollectionItem;
+import org.netxms.client.datacollection.DataCollectionObject;
 import org.netxms.client.objects.AbstractObject;
+import org.netxms.client.objects.Container;
 import org.netxms.client.objects.GenericObject;
 import org.netxms.client.objects.Node;
+import org.netxms.client.objects.configs.CustomAttribute;
 import org.netxms.tests.TestConstants;
 
 public class TestHelper
 {
+   public static String TEST_CONTAINER_A = "A-container";
+   public static String TEST_CONTAINER_B = "B-container";
+   public static String TEST_CONTAINER_C = "C-container";
+   public static String TEST_CONTAINER_V = "V-container";
+   public static String TEST_CONTAINER_X = "X-container";
```

Comment:
Same constants defined in different files. Purpose of constant is that it is defined once. I propose to provide String[] containers array from the caller class. 
Also constants that are not used outside of class should not be public.